packages/apps/transfer/src/pages/faucet/existing/index.tsx

```diff
@@ -64,7 +78,23 @@ const ExistingAccountFaucetPage: FC = () => {
       setRequestStatus({ status: 'processing' });
 
       try {
-        await fundExistingAccount(accountName, chainID, AMOUNT_OF_COINS_FUNDED);
+        const result = (await fundExistingAccount(
+          accountName,
+          chainID,
+          AMOUNT_OF_COINS_FUNDED,
+        )) as IFundExistingAccountResponse;
```

Comment:
It was already here so I understand you just went with it, but this `useCallback` + async/await flow + error handling is not very idiomatic React.

Yet the `as` casting, any chance `fundExistingAccount` could be typed properly?

Reply:
Will write this down for another PR. Attempted to fix this but it requires a lot of refactoring